### ISSUE TITLE
Do you know why the performance gap is so big between validation and test set?

### ISSUE THREAD

**Issue by xizaoqu** [edwardzhou130/Panoptic-PolarNet#11]

**Comment by edwardzhou130** [comment]
1. I used the same best pretrained model in the validation set to generate the test set result. I did not retrain the network with both train/val data or use TTA, both of which could improve the test set result by a lot (Used by many other methods in their test set submissions).
2. The data distribution in semanticKITTI val and test sets are very big. You will find big result differences in most of the classes, especially those rare classes like motorcyclist.
